sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackendWelding.kt

```diff
@@ -23,6 +24,8 @@ internal interface RustBackendWelding {
 
     val saplingParamDir: File
 
+    suspend fun initBlockMetaDb(): Int
```

Comment:
What is the int return value?

Reply:
According to `lib.rs` implementation: 
`/// Returns 0 if successful, or -1 otherwise.`
But maybe we could have true/false values instead, as we have in case of `initBlocksTable`? @str4d

Reply:
````
 static func initBlockMetadataDb(fsBlockDbRoot: URL) throws -> Bool {
        let blockDb = fsBlockDbRoot.osPathStr()

        let result = zcashlc_init_block_metadata_db(blockDb.0, blockDb.1)

        guard result else {
            throw lastError() ?? .genericError(message: "`initAccountsTable` failed with unknown error")
        }

        return result
    }
````

Reply:
As Carter pointed out recently, the Rust layer error handling and possibly switching to UniFFI will be done. So this change could be part of it then.